src/component/association-select.js

```diff
@@ -51,6 +53,15 @@ export class AssociationSelect {
     this.entityManager  = entityManager;
   }
 
+  validTypes = ['checkboxes', 'select'];
+
+  @computedFrom('type')
+  get view() {
+    return this.validTypes[this.type]
+      ?  `./${this.type}.html`
+      : './select.html';
```

Comment:
I think you meant `this.type in this.validTypes`.

Because `this.validTypes["select"]` and `this.validTypes["checkboxes"]` will always be `undefined`.